### PR TITLE
Added inquiry status on ui

### PR DESCRIPTION
**Pull request by keshavgr8**
Finishes story [#155031810](https://www.pivotaltracker.com/story/show/155031810)

### CODE REVIEW

**Comment by aayushKhandpur** [comment]
@keshavgr8 
Use drop down for UX. This should be updatable by the counsellor. Should be only available on counsellor module. 
Also, include modal ui on "Close". If that is part of this story

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review comments

**Comment by keshavgr8** [comment]
Comments addressed in commit 75da801

File: client-src/src/pages/inq-details/inq-details.html [hunk 1]
```diff
@@ -13,7 +13,15 @@
...
-      <ion-item-divider color="light">Personal Details<span class="inq-status" *ngIf="currentInq">Status: {{currentInq.data.inquiryStatus}}</span></ion-item-divider>
+      <ion-item-divider color="light">Personal Details
+        <span class="inq-status" *ngIf="currentInq">
+          Status:
+          <select (change)="changeInqStatus($event)">
+            <option [selected]=isInqOpen(currentInq.data.inquiryStatus) value="open">Open</option>
+            <option [selected]=!isInqOpen(currentInq.data.inquiryStatus) value="close">Closed</option>
+          </select>
+        </span>
+      </ion-item-divider>
```
Comment (aayushKhandpur) [inline client-src/src/pages/inq-details/inq-details.html]: share updated screen shot

Comment (aayushKhandpur) [reply client-src/src/pages/inq-details/inq-details.html]: Make this a little wider. Also include closing_status, closing_substatus

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review

File: client-src/src/pages/inq-details/inq-details.html (hunk 1 above)
Comment (keshavgr8) [reply client-src/src/pages/inq-details/inq-details.html]: resolved in commit 95cfd46

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**